### ISSUE TITLE
-Wenum-conversion in drivers/regulator/lp872x.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1481]
```
drivers/regulator/lp872x.c:689:57: error: implicit conversion from enumeration type 'enum lp872x_dvs_state' to different enumeration type 'enum gpiod_flags' [-Werror,-Wenum-conversion]
        dvs->gpio = devm_gpiod_get_optional(lp->dev, "ti,dvs", pinstate);
                    ~~~~~~~~~~~~~~~~~~~~~~~                    ^~~~~~~~
1 error generated.
```

Patch submitted: https://lore.kernel.org/r/20211019004335.193492-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Accepted: https://git.kernel.org/broonie/regulator/c/061514dbfb79910ef60eb40dd9fc528be3f45d62

**Comment by nathanchance** [comment]
Fixed in next-20211025.
